contracts/argent_account.cairo

```diff
@@ -118,4 +145,69 @@ mod ArgentAccount {
         }
         ecdsa::check_ecdsa_signature(hash, guardian_backup::read(), signature_r, signature_s)
     }
+
+    #[external]
+    fn trigger_escape_signer() {
```

Comment:
seems like in the 0.x contracts the external methods are `camelCase`. should we do the same here @julien?